src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -88,17 +88,24 @@ class NativeSessionStorage implements SessionStorageInterface
      */
     public function __construct(array $options = array(), \SessionHandlerInterface $handler = null, MetadataBag $metaBag = null)
     {
-        $options += array(
-            // disable by default because it's managed by HeaderBag (if used)
-            'cache_limiter' => '',
-            'use_cookies' => 1,
-        );
+        $this->setMetadataBag($metaBag);
+
+        if (\PHP_SESSION_ACTIVE === session_status()) {
+            if (!empty($options) || null !== $handler) {
+                throw new \LogicException('Cannot change options or handler of an active session');
```

Comment:
Do we really to throw here ? maybe a test should be added to test this behaviour ?